backend/src/packages/users/user.repository.ts

```diff
@@ -14,15 +15,29 @@ class UserRepository implements IRepository {
   }
 
   public async findAll(): Promise<UserEntity[]> {
-    const users = await this.userModel.query().execute();
+    const users = await this.userModel
+      .query()
+      .withGraphFetched('userDetails')
+      .returning('*')
```

Comment:
`withGraphFetched` under the hood would make subselects on `userDetails` table for each user, instead of joining this relation in single query
`withGraphJoined` would make a join for mentioned relation
Sooo for select queries it's better to use `withGraphJoined` (or `joinRelated`) - `withGraphFetched` may be used for inserts\updates (as in this case we won't be able to make direct join)
```suggestion
      .query()
      .select()
      .withGraphJoined('userDetails')
```